source/features/hide-diff-signs.css

```diff
@@ -2,3 +2,13 @@
 .rgh-hide-diff-signs .blob-code-marker-cell::before { /* Commit page */
 	visibility: hidden;
 }
+
+/*
+
+File URLs
+
+Any commits on https://github.com/sindresorhus/refined-github/commits/main
+Any reviewed block on https://github.com/sindresorhus/refined-github/pull/1783
+Full diff on https://github.com/sindresorhus/refined-github/pull/1783/files
+
+*/
```

Comment:
Same goes for `hide-diff-signs` cc @FloEdelmann, hidden everywhere. 
<img width="380" alt="Screenshot 27" src="https://user-images.githubusercontent.com/1402241/215486483-f04ca80c-4da4-4ad0-bd43-db01647b63a6.png">

<img width="566" alt="Screenshot 29" src="https://user-images.githubusercontent.com/1402241/215486473-c3f31088-2324-4350-815d-811478de0c8e.png">


<img width="441" alt="Screenshot 28" src="https://user-images.githubusercontent.com/1402241/215486451-799807ca-6ed9-402a-ac9d-69607563d0a0.png">

Reply:
@FloEdelmann never mind, I reread your report. Will fix separately.

Reply:
1. Go to https://github.com/refined-github/refined-github/edit/main/package.json
2. edit anything
3. click on preview

Diff signs are not hidden there:

![grafik](https://user-images.githubusercontent.com/202916/215487094-10a6a89f-d4d6-4900-aee7-8a66d51a3de7.png)

Reply:
See https://github.com/refined-github/refined-github/pull/6255